exploration/text-vs-code.md

```diff
@@ -136,16 +209,93 @@ The current syntax includes some plain-ascii keywords:
+such as `:number`, `$var`, `|literal|`, `+bold`, `-bold`, and posibly `@attr`.
 
 The current syntax supports unquoted literal values as operands.
 
-Messages themselves are "simple strings" and must be considered to be a single
-line of text. In many containing formats, newlines will be represented as the local
-equivalent of `\n`.
+Messages themselves are "simple strings" and must be considered to be WYSIWYG.
+The WYSIWYG nature of representing a message pattern is independent of whether the message is a single line or contains multiple lines.
+
+There is no restriction that a message must only contain a single line (that is, not contain any newline characters),
+nor are there constraints about how newlines must be represented. As our [`spec/syntax.md`](../spec/syntax.md) states:
+
+> Any Unicode code point is allowed, except for surrogate code points U+D800 through U+DFFF inclusive.
+
+> Whitespace in _text_, including tabs, spaces, and newlines is significant and MUST be preserved during formatting.
+
+> ... Instead, we tolerate direct use of nearly all
+characters (including line breaks, control characters, etc.) and rely upon escaping
+in those outer formats to aid human comprehension (e.g., depending upon container
+format, a U+000A LINE FEED might be represented as `\n`, `\012`, `\x0A`, `\u000A`,
+`\U0000000A`, `&#xA;`, `&NewLine;`, `%0A`, `<LF>`, or something else entirely).
+
+## Simple Messages
+
+In the Subcommittee meetings following Github discussions on Issues #493 and #499,
+the general consensus that formed for simple messages
+is that we allow them to be unquoted.
+
+("Simple messages" refers to messages consisting solely of a pattern, and thus are not complex messages.)
+
+Because the simple message pattern consists of the entire message,
+the pattern includes any leading or trailing whitespace.
+
+Given simple messages already being decided at a high level,
+the design decisions below for the proposed and alternative designs pertain specifically to complex messages.
 
 ## Proposed Design
 
+### Start in text, encapsulate message, always quote patterns
+
+Description:
+
+Since simple messages are unquoted (starting in text mode),
+complex messages must also start in text mode.
+
+Within a complex message, patterns are always quoted with `{{...}}` or other choice of delimiter.
+
+The entire complex message is also wrapped with `{{...}}` or other choice of delimiter.
+This allows interior "code mode" of message to have flexible whitespace in between tokens
+and _around_ quoted patterns.
+
+Pros
+
+* The rule about the whether leading and trailing whitespace is included is simple and unambiguous.
+* This matches the WYSIWIG behavior that simple messages preserve.
+* The patterns can be detected within the pattern more easily due to the delimiters serving as a visual anchor.
+* Requiring all patterns to be quoted minimizes the number of characters that need to be escaped within a pattern to 3:
```

Comment:
I think this also holds for at least some of the other proposals.

In fact, if we always quote variant patterns, we must make the _closing_ delimiter special. Theoretically, in other proposals, we could only special-case the opening delimiter, because both code and placeholders would be wrapped in the same delimiter. (Although that would require agreeing to a different way of preserving whitespace than the currently agreed `{{ ... }}`.)

Reply:
> In fact, if we always quote variant patterns, we must make the closing delimiter special.

But we have to do it for all the other proposals too.
Because all allow for optional wrapping.